code/scott/python/oop/bank.py

```diff
@@ -0,0 +1,31 @@
+from types import ClassMethodDescriptorType
```

Comment:
I don't think we this this here

Reply:
VS Code added that automatically so I left it in.